frontend/src/views/sessions/index.tsx

```diff
@@ -10,13 +10,36 @@ import {
 } from "../../components/action-buttons";
 import { ContentArea } from "../../components/layout";
 import { FaPlus } from "react-icons/fa";
+import { Session } from "../../api/defs/types";
 
 const ConnectedSessionList = connect((state) => ({
```

Comment:
I'm wondering why this is using the `connect` style and not the `useSelector/useDispatch` style...

Reply:
'cause it's old and I didnt' change it